release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -605,6 +620,16 @@ revision "2023-06-30" {
         "RFC 4862: IPv6 Stateless Address Autoconfiguration";
     }
 
+    leaf unsolicited-na-accepted {
+      type boolean;
```

Comment:
I would question whether this should be boolean.

@cagdasalagoz-arista's implementation does it this way; 
in srlinux we have an implementation that offers greater flexibility and uses [a enum instead](https://github.com/nokia/srlinux-yang-models/blob/6dd7b5a91433966dc9e4b763389a1d090355771b/srlinux-yang-models/srl_nokia/models/interfaces/srl_nokia-interfaces-nbr.yang#L342-L360).

Obviously from our point of view the enum is a better option; but considering that there are only 2 implementations I'm not sure what is the procedure to decide / break the tie here. @dplore any thoughts?

Reply:
I do like the enum in this case.  It does not seem overly complex.  Vendors may only support a subset of the options.

@cagdasalagoz-arista can you pelase update similar to the following?   I checked the arista doc and it looks like it also implements "none" as the default, so I think we can leave that in.

```
      leaf learn-unsolicited {
        type enumeration {
          enum none {
            value 0;
          }
          enum global {
            value 1;
          }
          enum link-local {
            value 2;
          }
          enum both {
            value 3;
          }
        }
        default "none";
        description
          "Sets if neighbors should be learned from unsolicited neighbor advertisements for global or link local addresses or both.";
      }

```